docs/fixture_usage.md

```diff
@@ -0,0 +1,148 @@
+# Guidelines for fixtures usage in OCS-CI
+
+## Basic information
+
+What is a pytest Fixture? You can find definition and examples documentation in
+[pytest web](https://docs.pytest.org/en/latest/fixture.html). Please read this
+documentation first.
+
+We should keep our fixtures organized and have all the dependencies well
+defined in fixtures.
+
+All the fixtures should be documented - documenting the return value is
+mandatory.
+
+Fixture is compose of its setup part, the code which is done for prepare
+resource and teardown part which is after yield of added via
+`request.addfinalizer` method. It's strictly forced to use `addfinalizer`
+method cause it can happen that what is after yield won't be proceed when
+exception will be thrown in the middle of the setup.
+
+The finalizer should be added as soon as possible to make sure the finalizer
+will be called at the end. (Otherwise we will hit the same issue as we have
+with yield)
+
```

Comment:
I would add paragraph about [usage of `contest.py`](https://docs.pytest.org/en/latest/fixture.html#conftest-py-sharing-fixture-functions) file and about ability to create it in each test directory and that fixtures contained in it can be used in all tests from all child directories. So there can be different conftest for manage tests and for ecosystem. In root directory can be conftest that contains fixtures for all tests.